### PR TITLE
DOC-5501: Added tracing page to docs

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/5.0/deploying-operating/observability/tracing.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Tracing Framework
+
+As a Corda Cluster Operator, you may want to configure sample rate when deploying the Helm chart. You can do this using the
+`--trace-samples-per-second` command line option that:
+
+* defaults to 1 sample per second if not set
+* allows specifying unsigned numeric values as samples per second
+* allows `unlimited` value to to sample all requests
+
+The tracing configuration supported by the Helm chart contains the following information in the `values.yaml` file:
```
Comment (josephzunigadaly) [inline content/en/platform/corda/5.0/deploying-operating/observability/tracing.md:22]: I'm told we haven't documented worker command line parameters. I think we should not document this one alone. Lets remove the command line parameter and focus on the Helm chart.

```suggestion
As a Corda Cluster Operator, you may want to configure a tracing server and sample rate when deploying the Helm chart. The tracing configuration supported by the Helm chart contains the following configuration in the `values.yaml` file:
```

File: content/en/platform/corda/5.0/deploying-operating/observability/tracing.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  corda5:
+    parent: corda5-cluster-observability
+    identifier: corda5-cluster-tracing
+    weight: 4000
+section_menu: corda5
+---
+
+# Tracing Framework
+
+As a Corda Cluster Operator, you may want to configure sample rate when deploying the Helm chart. You can do this using the
```
Comment (yarivus) [inline content/en/platform/corda/5.0/deploying-operating/observability/tracing.md:15]:
```suggestion
As a Corda Cluster Operator, you may want to configure tracing sample rate when deploying the Helm chart. You can do this using the
```

**Review by yarivus — CHANGES_REQUESTED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by josephzunigadaly — APPROVED** [review]

File: content/en/platform/corda/5.0/deploying-operating/observability/tracing.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+# Tracing Framework
+
+As a Corda Cluster Operator, you may want to configure a tracing server and sample rate when deploying the Helm chart.
+The tracing configuration supported by the Helm chart contains the following configuration in the `values.yaml` file:
+
+```
+tracing:
+  # -- URL for endpoint to send Zipkin-format distributed traces to e.g. http://tempo:9411
```
Comment (amiegrace12) [inline content/en/platform/corda/5.0/deploying-operating/observability/tracing.md:20]:
```suggestion
  # -- URL for the endpoint to send Zipkin-format distributed traces to for example http://tempo:9411
```

File: content/en/platform/corda/5.0/deploying-operating/observability/tracing.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  endpoint: ""
+  # --  Number of request traces to sample per second, defaults to 1 sample per second. Set to 'unlimited' to record all
+  traces, but in this case amount of tracing data produced can be quite vast.
```
Comment (amiegrace12) [inline content/en/platform/corda/5.0/deploying-operating/observability/tracing.md:23]:
```suggestion
  traces, but in this case the amount of tracing data produced can be quite vast.
```

**Review by amiegrace12 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @AnnaK-R3

**Pull request merged**